test/parse-markdown-table/valid-tables.test.ts

```diff
@@ -180,3 +180,23 @@ describe('createMarkdownArrayTable', () => {
     expect(rows).toMatchSnapshot()
   })
 })
+
+describe('only two dashes', () => {
+  async function setup() {
+    const text = `
+      name | age
+      -- | --
+      Toby | 3
+      Alex | 25
+    `
```

Comment:
For sake of consistency, please you data from this table: https://github.com/ksxnodeapps/fun-little-utilities/blob/80a3fac2ef9963e0ea83ef87d2c611acbdf9168d/test/parse-markdown-table/valid-tables.test.ts#L160-L164

Reply:
Ok I will change the data